Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
+                    },
+                    "continue": {
+                      "type": "string",
+                      "description": "%c_cpp.configuration.commentContinuationPatterns.items.anyof.object.continue.description%"
+                    }
                   }
                 }
-              }
-            ]
+              ]
+            },
+            "uniqueItems": true,
+            "description": "%c_cpp.configuration.commentContinuationPatterns.description%",
+            "scope": "window"
           },
-          "scope": "resource"
-        },
-        "C_Cpp.debugShortcut": {
-          "type": "boolean",
-          "default": true,
-          "description": "%c_cpp.configuration.debugShortcut.description%",
-          "scope": "resource"
-        },
-        "C_Cpp.legacyCompilerArgsBehavior": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "%c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription%",
-          "scope": "resource"
-        },
-        "C_Cpp.sshTargetsView": {
-          "type": "string",
-          "enum": [
-            "enabled",
-            "disabled",
-            "default"
-          ],
-          "default": "default",
-          "description": "%c_cpp.configuration.sshTargetsView.description%",
-          "scope": "window"
+          "C_Cpp.configurationWarnings": {
+            "type": "string",
+            "enum": [
+              "enabled",
+              "disabled"
+            ],
+            "default": "enabled",
+            "description": "%c_cpp.configuration.configurationWarnings.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.intelliSenseCachePath": {
+            "type": "string",
+            "markdownDescription": "%c_cpp.configuration.intelliSenseCachePath.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.intelliSenseCacheSize": {
+            "type": "number",
+            "default": 5120,
+            "markdownDescription": "%c_cpp.configuration.intelliSenseCacheSize.markdownDescription%",
+            "scope": "machine-overridable",
+            "minimum": 0
+          },
+          "C_Cpp.intelliSenseMemoryLimit": {
+            "type": "number",
+            "default": 4096,
+            "markdownDescription": "%c_cpp.configuration.intelliSenseMemoryLimit.markdownDescription%",
+            "scope": "machine-overridable",
+            "minimum": 256,
+            "maximum": 16384
+          },
+          "C_Cpp.intelliSenseUpdateDelay": {
+            "type": "number",
+            "default": 2000,
+            "description": "%c_cpp.configuration.intelliSenseUpdateDelay.description%",
+            "scope": "application",
+            "minimum": 500,
+            "maximum": 3000
+          },
+          "C_Cpp.updateChannel": {
```

Comment:
@Colengms is it ok to remove this setting now?

Reply:
I believe so.